src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1385,10 +1384,15 @@ private static void validateViolationSection(String fileName, String sectionName
                 .isEqualTo("");
         }
         else {
+            final String subsectionTextContent = subSection.getTextContent()
+                    .replaceAll("\n\\s+", "\n")
+                    .replaceAll("\n", " ")
```

Comment:
Can we condense this into one `replaceAll` call, or am I missing something? `\n\\s+` -> `\n` -> ` `?